frappe/core/doctype/docfield/docfield.json

```diff
@@ -192,7 +192,7 @@
   },
   {
    "default": "0",
-   "depends_on": "eval:['Data', 'Select', 'Text', 'Small Text', 'Text Editor'].includes(doc.fieldtype)",
+   "depends_on": "eval:['Data', 'Select', 'Text', 'Small Text', 'Text Editor', 'Link', 'Dynamic Link'].includes(doc.fieldtype)",
```

Comment:
@hrwx this is no longer needed, right? (In case it is, we'd need to update the `modified` timestamp.)